### ISSUE TITLE
Analysis packages cannot be unreleased due to an incorrect ID in the project cockpit

### ISSUE THREAD

**Issue by AndyDaniel1** [dzhw/metadatamanagement#3110]
In the project cockpit analysis packages are managed with an ID that looks like a datapackage ID (stu-XXX$) --> first screenshot. After the release, however, the analysis package receives a different ID (ana-XXX$) --> second screenshot. 

Thus the cockpit does not find the analysis package any more and reports the status that the package is not published. Due to this, an unrelease is not possible. 

Screenshot 1
![grafik](https://user-images.githubusercontent.com/17981185/183854440-f1330c1c-88a5-4e50-9f68-795eabd17523.png)

Screenshot 2
![grafik](https://user-images.githubusercontent.com/17981185/183854605-685aaffd-6a3f-4d18-a049-6fe284f64c6b.png)

**Comment by AndyDaniel1** [comment]
Tasks:
- Implementation of the correct id assignment in the project cockpit
- Correction of the wrong id in the database (stu-goldan2022$ --> an-goldan2022$)

**Comment by cschwartze** [comment]
There was no distinction between two cases Data/Analysis Package in the cockpit.

Hence the generated URLs were always stu-xxx and only after updating the corresponding template at https://github.com/dzhw/metadatamanagement/blob/cschwartze/issue3110/src/main/webapp/scripts/dataacquisitionprojectmanagement/directives/project-cockpit-versions.html.tmpl, the URLs for all package types generated correctly (and shadow copies are displayed).

**Comment by tilovillwock** [comment]
Reopening since this has neither been deployed nor tracked in a milestone yet.

**Comment by AndyDaniel1** [comment]
a final test can only be done on the prod system. Therefore I keep this issue open but it can be merged into the prod system. 

The test label is removed for now.
